server/src/database/config/fakeData.sql

```diff
@@ -0,0 +1,41 @@
+
+INSERT INTO course(category_id,title,author_name,url,rate,reviews,description, source)
+VALUES(
+    '1',
+    'computer',
+    'Moh',
+    'www.course1.com',
```

Comment:
the url must be unique
change the url each row
for example
`www.course1.com/1`
`www.course1.com/2`
`www.course1.com/3`